src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheckTest.java

```diff
@@ -115,9 +115,6 @@ public void testTags() throws Exception {
             "78:8: " + getCheckMessage(MSG_UNUSED_TAG, "@param", "Unneeded"),
             "79: " + getCheckMessage(MSG_UNUSED_TAG_GENERAL),
             "87:8: " + getCheckMessage(MSG_DUPLICATE_TAG, "@return"),
-            "109:23: " + getCheckMessage(MSG_EXPECTED_TAG, "@param", "aOne"),
-            "109:55: " + getCheckMessage(MSG_EXPECTED_TAG, "@param", "aFour"),
-            "109:66: " + getCheckMessage(MSG_EXPECTED_TAG, "@param", "aFive"),
```

Comment:
Check originally reported parameters with missing descriptions but now with new changes we won't even get a warning that the descriptions are missing. I personally not a fan in this lost of functionality but this is similar behavior with `@throws` tag. Since we feel this check does too much already, this type of functionality should probably be in a separate check.

@romani Are you ok with this?

Reply:
@rnveach after implementing  #6746 this warning will be shown again.

Reply:
After thinking for a while ...
I think it is ok to have such regression, it is good that it is false-negative, that is good.
It is good to validate params that are referenced in javadoc, at least user tried to explain it, so it is good help him to it properly.
in #6746, it should be configurable to report on missed parameters, as most of parameters are self-explaining, so it is ok to skip for some time.